CONTRIBUTING.md

```diff
@@ -1,9 +1,17 @@
 # Contributing
 
-Thank you for considering to contribute to `pacdef`. The recommended workflow is this:
+## General Steps
+
+Thank you for considering to contribute to `pacdef`. The recommended workflow is
+this:
 
 1. Open a github issue, mention that you would like to fix the issue in a PR.
 2. Wait for approval.
 3. Fork the repository and implement your fix / feature.
 4. Make sure your code generates no warnings, and passes `rustfmt` and `clippy`.
 5. Open the pull request.
+
+## Rust-Analyzer Issues
+
+Rust Analyzer may not work unless both the `pacutils` and `apt` packages are
```

Comment:
Are you using vscode? I tested the repo in vscode and rust-analyzer works fine but it crashes in neovim which is my main editor unless those packages are installed. Although I haven't tested it thoroughly so it may just be me imagining things.

Reply:
I'm using neovim with coc-rust-analyzer.

Reply:
Hmm, neovim but nvim-lsp, I wonder if that's making a difference. I guess I am also using nightly rust-analyzer.